### ISSUE TITLE
Stone Tools useless?

### ISSUE THREAD

**Issue by G-Stav** [oitsjustjose/Natural-Progression#11]
With version 1.14.4 1.2.0 stone tools have mirrored wooden tools and cannot be used to mine blocks. This may also maybe be breaking Waterstrainers shovel the garden trowel. 

Maybe it's possible to add a whitelist for which tools do disable instead?

**Comment by frakier** [comment]
confirmed....
it is this bit of code... in com.oitsjustjose.naturalprogression.common.config; lines 61-63
REMOVE_STONE_TOOL_RECIPES = COMMON_BUILDER.comment(
                "Setting this to true prevents the ability to craft stone tools. This is totally unrealistic anyways.")
.define("removeWoodenToolRecipes", true);
line 63 should read
.define("removeStoneToolRecipes", true);

**Comment by oitsjustjose** [comment]
Thanks for the insight! Fixed this issue in e099b785bdde288393733464cba5606195f77aea.

**Comment by frakier** [comment]
There still may be more to the problem, I changed it and ran the code and still get the message in red on the stone tools. Digging some more.

**Comment by frakier** [comment]
added to the ToolTips.java file...
`public ToolTips()
    {
        if (CommonConfig.REMOVE_WOODEN_TOOL_RECIPES.get())
        {
            neuteredTools.add(Items.WOODEN_AXE);
            neuteredTools.add(Items.WOODEN_PICKAXE);
            neuteredTools.add(Items.WOODEN_SHOVEL);
            neuteredTools.add(Items.WOODEN_HOE);
            neuteredTools.add(Items.WOODEN_SWORD);
        }
        if (CommonConfig.REMOVE_STONE_TOOL_RECIPES.get())
        {
            neuteredTools.add(Items.STONE_AXE);
            neuteredTools.add(Items.STONE_PICKAXE);
            neuteredTools.add(Items.STONE_SHOVEL);
            neuteredTools.add(Items.STONE_HOE);
            neuteredTools.add(Items.STONE_SWORD);
        }`

made this change as well on ToolTips.java...
`private ArrayList<Item> neuteredTools = Lists.newArrayList();
    //Items.WOODEN_AXE, Items.WOODEN_PICKAXE, Items.WOODEN_SHOVEL, Items.WOODEN_HOE, Items.WOODEN_SWORD
    //Items.STONE_AXE, Items.STONE_PICKAXE, Items.STONE_SHOVEL, Items.STONE_HOE, Items.STONE_SWORD`

changed ToolNeutering.java line26...
`ToolItem tool = (ToolItem) heldItem.getItem();
            //if (tool.getTier() == ItemTier.WOOD || tool.getTier() == ItemTier.STONE)
            if ((CommonConfig.REMOVE_WOODEN_TOOL_RECIPES.get() && tool.getTier() == ItemTier.WOOD) ||
                    (CommonConfig.REMOVE_WOODEN_TOOL_RECIPES.get() && tool.getTier() == ItemTier.WOOD))`

and line 36 {36 after above change}...
`SwordItem tool = (SwordItem) heldItem.getItem();
            //if (tool.getTier() == ItemTier.WOOD || tool.getTier() == ItemTier.STONE)
            if ((CommonConfig.REMOVE_WOODEN_TOOL_RECIPES.get() && tool.getTier() == ItemTier.WOOD) ||
                    (CommonConfig.REMOVE_WOODEN_TOOL_RECIPES.get() && tool.getTier() == ItemTier.WOOD))
            {
                event.setCanceled(true);
            }`

and of cahnge to the CommonConfig.java

**Comment by oitsjustjose** [comment]
Yeah, so I didn't really think through a lot of this stuff at the time - there was a large gap between when I started and finished that update.

I've re-worked the tooltips to take into account the same exact logic as the event itself, so all stone-tier items show the right tooltip if stone tools are disabled, and all wood-tier items show the right tooltip if wooden tools are disabled.

I've also fixed up the logic in the events to take into account the different configs for the different `ToolTiers`, so that should fix that issue as well. I'm about to test to make sure everything works as expected.
